spec/requests/api/collections_spec.rb

```diff
@@ -134,6 +134,11 @@ def test_collection_bulk_query(collection, collection_url, klass, id = nil)
       test_collection_query(:hosts, hosts_url, Host, :guid)
     end
 
+    it 'queries NetworkRouters' do
```

Comment:
@imtayadeway, sure although I believe all those test could go in the collection but that would be the purpose of another PR ;)